### DISCUSSION TITLE
Vents & Holes

### DISCUSSION THREAD

**Discussion by FireStudent321** [firemodels/fds#13067] [General]
Hi,

I know that an obstruction gets shifted to the nearest cell using FDS if the dimensions have not been rounded up or down already. But does the vent or the hole get shifted to the nearest cell as well using FDS or is it independent of the cell size?

Also, I was just wondering why the code does not use holes for room 1 and room 3. Link to the code: https://github.com/firemodels/fds/blob/11bce1b24a3640876b27751b8fd216327d5c30e8/Validation/NBS_Multi-Room/FDS_Input_Files/NBS_100Z.fds

Thanks.

**Comment by drjfloyd** [comment]
VENTs also snap to the nearest grid cell.

For the inputs problaby no particular reason. When you specify a HOLE, FDS simply makes a new set of OBST like the ones seen in the input file.. These files date back two decades, so if there was some reason at the time, that reason is probably lost to history.

**Reply by FireStudent321** [reply]
Thanks, drjfloyd.

So what you are saying is that VENTS (&VENT) or HOLES (&HOLE) also get shifted like the obstructions (&OBST) to the nearest grid cell? So if the top part of the vent or hole is 0.987, it gets rounded or shifted up to 1, if the cell is 0.1 high? So both holes and vents are cell size dependent?

**Reply by drjfloyd** [reply]
VENT, OBST, and HOLE will all snap to the grid.
